bigip.go

```diff
@@ -213,7 +213,7 @@ func (b *BigIP) delete(path ...string) error {
 }
 
 func (b *BigIP) post(body interface{}, path ...string) error {
-	marshalJSON, err := jsonMarshal(body)
+	marshalJSON, err := json.Marshal(body)
```

Comment:
The special escape processing https://github.com/scottdware/go-bigip/blob/master/bigip.go#L304 seems to be intended here. There are a number of other instances below where we might need to use ```jsonMarshal```
@scottdware Please comment. Thanks